packages/component-utils/src/componentStyleReset.js

```diff
@@ -16,6 +16,7 @@
 
 /* @flow */
 export default (theme: { [string]: string } = {}) => ({
+  all: 'initial',
```

Comment:
We had this in here originally, but then Kyle removed it.  Why?  Do we want it or not?  Also, it breaks the "Icons by category" demo.